01-Login/app.js

```diff
@@ -74,6 +68,12 @@ function runFunction ($rootScope, auth, store, jwtHelper, $location){
         $location.path('/login');
       }
     }
-
   });
+
+  $rootScope.logOut = function () {
+    store.remove('profile');
+    store.remove('token');
+    auth.signout();
```

Comment:
@ntotten first I added the logout here, as it make the example more clear/clean.
I also added this line (auth.signout), although @christiannwamba had not used it, I don't know why, perhaps there is a reason.